apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SubscriberClientTest.cs

```diff
@@ -1070,43 +1070,72 @@ public void ValidParameters()
         }
 
         [Fact]
-        public void SilentlyFixedParameters()
+        public void MinimumDelayIsUsedWhenMinimumAckDeadlineIsSpecified()
         {
             var subscriptionName = new SubscriptionName("project", "subscriptionId");
             var clients = new[] { new FakeEmptySubscriberServiceApiClient() };
 
-            // Test MinimumLeaseExtensionDelay is honoured when minimum ack-deadline used.
-            var settingsAckExtension1 = new SubscriberClient.Settings
+            // Test that MinimumLeaseExtensionDelay is honoured when minimum ack-deadline is specified.
```

Comment:
Given the confusion we've had, let's rename this to "when the ack-deadline is specified with the minimum possible value".

Reply:
Done.